adi/jesd_internal.py

```diff
@@ -5,7 +5,94 @@
 from .sshfs import sshfs
 
 
-class jesd:
+class eye_scan(object):
+    _jesd_es_duration_ms = 100
+    _jesd_prbs = 7
+
+    _half_rate = {"mode": "Half Fate", "scale": 0.004}
```

Comment:
Let's scale to mV for half rate the API already scales things.
So here we use 1.0

Reply:
Fixed